.travis.yml

```diff
@@ -115,10 +115,10 @@ install:
         && cd occa && git reset --hard 327badb && cd ..
         && make -C occa -j2
         && export OCCA_DIR=$PWD/occa;
-# LIBXSMM v1.15 (need to use specific commit, XSMM develops on master)
+# LIBXSMM v1.16 (need to use specific commit, XSMM develops on master)
   - if [[ "$TRAVIS_CPU_ARCH" == "amd64" ]]; then
         git clone https://github.com/hfp/libxsmm.git
-        && cd libxsmm && git reset --hard 724205c && cd ..
+        && cd libxsmm && git reset --hard 6d6a0f6 && cd ..
```

Comment:
Just curious. Would it be the same if we used `git checkout tags/1.16` here instead of  `git reset --hard <commit>`, or we wouldn't want to do a `checkout`?

Reply:
`git checkout -f <commit>` would be good.

Reply:
It would have the same effect, I believe. We've just been using commit numbers because sometimes we want a commit between releases (like with OCCA and PETSc).

Reply:
Updated. If there are no objections, I can merge now. It annoys me that our CI fails due to two external issues, but I don't suppose we can do more about that than we are doing right now.

Reply:
Difference is that `checkout` (or `switch`) changes `HEAD` while `reset` changes the index/working tree. The difference arises if you were to commit something or otherwise care about that detail.

In the interest of reproducible CI and ability to maintain a `stable` branch (post-1.0 if not before), we should probably move toward pinning versions of each dependency.